alacritty/src/input.rs

```diff
@@ -1171,6 +1171,7 @@ mod tests {
                 };
 
                 assert_eq!(processor.ctx.mouse.click_state, $end_state);
+                assert_eq!(processor.ctx.mouse.last_click_button, $end_button);
```

Comment:
What's the goal of this? Was there a reason that triggered this addition?

Reply:
The input state included this and it is part of the double- and triple-click logic, so it seemed good to check it as part of the ending state.  I can remove it if you'd like.